make.bat

```diff
@@ -1,17 +1,14 @@
 @ECHO OFF
```

Comment:
this is an auto-generated file, so I'm hesitant to edit it -- what are these changes doing/fixing? are they fixed in the latest version of sphinx? maybe we could upgrade our sphinx setup

Reply:
On second thought, this is just navel gazing and if it makes it work for you on Windows then let's go ahead.

Reply:
I don't know if it is autogenerated or not. But I needed to change it so that it worked with parameters. The file as it is does not expect parameters, just which type of manual to generate.